src/session/sidebar/row.rs

```diff
@@ -347,11 +350,11 @@ impl Row {
     }
 }
 
-fn stringify_message(message: Message) -> String {
+fn stringify_message(message: Message, is_own_chat: bool) -> String {
     let mut show_sender = match message.chat().type_() {
         ChatType::BasicGroup(_) => true,
         ChatType::Supergroup(supergroup) => !supergroup.is_channel(),
-        ChatType::Private(_) | ChatType::Secret(_) => message.is_outgoing(),
+        ChatType::Private(_) | ChatType::Secret(_) => !is_own_chat && message.is_outgoing(),
```

Comment:
We always have to show the sender in case the message is outgoing, otherwise messages forwarded in the 'Saved Messages' will not show the sender in the sidebar.

Reply:
I have added the `MessageForwardInfo` for that. But there is one point where I am not satisfied. I left a `TODO` there. Maybe you have an idea.
In this turn, I cleaned up the `stringify_message` function in `row.rs`, which has become quite confusing. I hope the readability is a bit improved now.

Reply:
I just realized the Android client does not show the sender at all. Even not for forwarded messages. However, the desktop client does.

Reply:
I am not quite sure, but it might be that the message is deleted. Thus, I turned the message field into a `WeakRef`.